docs/API.md

```diff
@@ -1702,10 +1703,29 @@ The above example is equivalent to the `meta/common-info` example above.
 
 **Not yet implemented**
 
-#### POST *id*/resources/name.stream
+#### GET *id*/meta/resources
```

Comment:
You are correct, it looks like this would be handled by the meta/any endpoint. Other sections that support the meta/any endpoint don't seem to refer to it in their section, so I think it's intended that meta/any be common knowledge. Agree, disagree?